scripts/patch-ha-addon-macvlan.sh

```diff
@@ -126,7 +126,7 @@ fi
 
 echo -n "RaspberryMatic Add-on Hostname (e.g. 5422eb72-raspberrymatic): "
 if [[ -z "${CCU_CONTAINER_NAME}" ]]; then
-  CCU_CONTAINER_NAME=$(docker ps --format '{{.Names}}' | grep "_raspberrymatic" | cut -c7-)
+  CCU_CONTAINER_NAME=$(docker ps --format '{{.Names}}' | grep "_raspberrymatic" | cut -c7- | awk '{print $0}')
```

Comment:
Sorry, but I think this change does not make sense. `CCU_CONTAINER_NAME` should already contain the full docker container name after the `cut -c7-` . Or what does the additional `awk '{ print $0}'` try to solve? Please elaborate.